dev/a11y_assessments/lib/use_cases/text_button.dart

```diff
@@ -44,7 +44,10 @@ class MainWidgetState extends State<MainWidget> {
               children: <Widget>[
                 const Text('This is a TextButton:'),
                 TextButton(
-                  onPressed: () {  },
+                  onPressed: () {
+                    ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
We should not introduce another widget in the sample unless necessary.

Do you know what native behavior is when clicking on a html button?

Reply:
Whenever you click an html button, there needs to be some type of action/function done whenever it is pressed/clicked. Since onPressed() does not have a callback function that does an action, this would be considered an a11y violation since the user is expecting the button to have some type of action.

In this case, I added a counter that increments up to let the user know that an action is happening. The screen reader will then update the user on the count every time it is clicked.

Side note: Since this isn't a toggle button, isSelected or aria-pressed is not necessary for this. Text button only calls a function, it does not change states from pressed to unpressed.